test/client/script.js

```diff
@@ -340,4 +340,77 @@ describe(`script cases`, () => {
             throw new Error(`Expected page type to be empty, got ${ getPageType() || 'undefined' } from ${ url }`);
         }
     });
+
+    it('should successfully get locale from script', () => {
+        const expectedLocale = 'en_US';
+
+        const url = insertMockSDKScript({
+            query: {
+                'locale': expectedLocale
+            }
+        });
+
+        const localeObject = getLocale();
+        const receivedLocal = `${ localeObject.lang }_${ localeObject.country }`;
+        if (expectedLocale !== receivedLocal) {
+            throw new Error(`Expected client id to be ${ expectedLocale }, got ${ receivedLocal } from ${ url }`);
+        }
+    });
+
+    it('should successfully get locale from browser settings', () => {
+        const expectedLocale = 'en_US';
+
+        // $FlowFixMe
+        Object.defineProperty(navigator, 'languages', {
```

Comment:
Do you need to do this?

Reply:
That didn't work since languages is read-only. I had to use `Object.defineProperty` and set `configurable:true` to get it set the languages in the test. I have modified it to remove the $FlowFixMe and make it maybe a little easier to read